docs/reference/content/guides/querying.md

```diff
@@ -0,0 +1,250 @@
+method instead of `field()` but it is used in much the same fashion.  `and()` and `or()` take a [`varargs`](https://docs.oracle
+.com/javase/8/docs/technotes/guides/language/varargs.html) parameter of type `Criteria` so you can include as many filters as necessary.
+
+
+## Text Searching
+
+Morphia also supports MongoDB's text search capabilities.  In order to execute a text search against a collection, the collection must 
+have a [text index]({{< docsref "/core/index-text/" >}}) defined first.  Using Morphia that definition would look like this:
+
+```java
+@Indexes(@Index(fields = @Field(value = "$**", type = IndexType.TEXT)))
+public static class Greeting {
+    @Id
+    private ObjectId id;
+    private String value;
+    private String language;
+    
+    ...
+}
+```
+
+The `$**` value tells MongoDB to create a text index on all the text fields in a document.  A more targeted index can be created, if 
+desired, by explicitly listing which fields to index.  Once the index is defined, we can start querying against it like this [test]({{< 
+srcref "morphia/src/test/java/org/mongodb/morphia/query/TestTextSearching.java">}}) does:
+
+```java
+morphia.map(Greeting.class);
+datastore.ensureIndexes();
+
+datastore.save(new Greeting("good morning", "english"));
+datastore.save(new Greeting("good afternoon", "english"));
+datastore.save(new Greeting("good night", "english"));
+datastore.save(new Greeting("good riddance", "english"));
+
+datastore.save(new Greeting("guten Morgen", "german"));
+datastore.save(new Greeting("guten Tag", "german"));
+datastore.save(new Greeting("guten Nacht", "german"));
+
+datastore.save(new Greeting("buenos días", "spanish"));
+datastore.save(new Greeting("buenas tardes", "spanish"));
+datastore.save(new Greeting("buenos noches", "spanish"));
+
+List<Greeting> good = datastore.createQuery(Greeting.class)
+                             .search("good")
+                             .order("_id")
+                             .asList();
+Assert.assertEquals(4, good.size());
+Assert.assertEquals("good morning", good.get(0).value);
+Assert.assertEquals("good afternoon", good.get(1).value);
+Assert.assertEquals("good night", good.get(2).value);
+Assert.assertEquals("good riddance", good.get(3).value);
+```
+
+As you can see here, we create `Greeting` objects for multiple languages.  In our test query, we're looking for occurrences of the work 
+"good" in any document.  We created four such documents and our query returns exactly those four.
+
+## Other Query Options
+
+There is more to querying than simply filtering against different document values.  Listed below are some of the options for modifying 
+the query results in different ways.
+
+### Projections
+
+[Projections]({{< docsref "reference/method/db.collection.find/#projections" >}}) allow you to return only a subset of the fields in a 
```

Comment:
http://docs.mongodb.org/manual/tutorial/project-fields-from-query-results/ might be a better link